src/components/SignUp.js

```diff
@@ -64,7 +70,7 @@ class SignUpForm extends Component {
     const validateEmail = () => {
       const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Maybe it would be better to use email validation library (or event overall data validation like `yup` or something simillar)